compiler/rustc_feature/src/active.rs

```diff
@@ -161,6 +161,9 @@ declare_features! (
     (active, staged_api, "1.0.0", None, None),
     /// Added for testing E0705; perma-unstable.
     (active, test_2018_feature, "1.31.0", None, Some(Edition::Edition2018)),
+    /// Allows non-`unsafe` —and thus, unsound— access to `Pin` constructions.
+    /// Marked `incomplete` since perma-unstable and unsound.
+    (incomplete, unsafe_pin_internals, "1.61.0", None, None),
```

Comment:
Looks like we're just in time for 1.60.0.

```suggestion
    (incomplete, unsafe_pin_internals, "1.60.0", None, None),
```